### PR TITLE
Feature/leaderboard

### PR DESCRIPTION
**Pull request by smalboeuf**
### In this PR
Added basic structure of leaderboard component
Styled some of the leaderboard (likely to change)

## To be done:
Complete compareByQuests.
More stylings.
Waiting for the implementation of Sequelize to connect it to the backend.

### CODE REVIEW

File: client/src/stories/leaderboard.stories.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+import Leaderboard from '../components/Leaderboard/Leaderboard';
+
+
+export default { title: 'Leaderboard' }
+
+export const leaderboard = () => <Leaderboard users={leaderboardData} />
+
+
+
+const leaderboardData = [
```
Comment (cawel) [inline client/src/stories/leaderboard.stories.js:11]: ⬆️ All that whitespace above that line tells me you're not using `Prettier` in your editor :)

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,2271 @@
+{
```
Comment (cawel) [inline package-lock.json:1]: I don't think there is a need for a `package-lock.json` at the project root. 

Rather, there should be one in each of those folders:
*  `client/`
* `back-end/`

File: client/src/stories/leaderboard.stories.js (hunk 1 above)
Comment (smalboeuf) [reply client/src/stories/leaderboard.stories.js:11]: its weird, it used to auto format on save, but it stopped working. Need to check it out again

File: client/src/components/Leaderboard/Leaderboard.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import "./Leaderboard.scss";
+
+
+export default function Leaderboard(props) {
```
Comment (cawel) [inline client/src/components/Leaderboard/Leaderboard.js:5]: I would not use a default export, but rather a named export:
```js
export const Leaderboard = (props) => {
```
but that remains a personal preference.

**Pull request merged**